mbed_tools_ci_scripts/license_files.py

```diff
@@ -15,54 +15,71 @@
 from datetime import datetime
 from mbed_tools_ci_scripts.utils.configuration import configuration, ConfigurationVariable
 from mbed_tools_ci_scripts.utils.logging import set_log_level, log_exception
+from mbed_tools_ci_scripts.utils.python_helpers import flatten_dictionary
 from pathlib import Path
 
 logger = logging.getLogger(__name__)
 
-LICENCE_HEADER_TEMPLATE = """Copyright (c){date}, {author}. All rights reserved.
+LICENCE_HEADER_TEMPLATE = """Copyright (c) {date}, {author}. All rights reserved.
 SPDX-License-Identifier: {licence_identifier}
 """
 
 
-class CopyrightNotifierToolWrapper:
-    """Wrapper over the licenseheaders tool.
+class LicenceHeader:
```

Comment:
Sorry I didn't mention this earlier but now we've added more static methods I wonder if this should be a class at all? Seems like we have a function that just calls some other functions, there's no invariant or state we're managing here.

Reply:
true
changed that all